lib/CL/pocl_image_util.c

```diff
@@ -255,6 +255,7 @@ pocl_get_image_information (cl_channel_order ch_order,
 /****************************************************/
 
 #define FOR4 unsigned i; for (i = 0; i < 4; i++)
+#define FOR2 unsigned i; for (i = 0; i < 2; i++)
```

Comment:
What is this macro trickery for? Pls add a comment above.

Reply:
AFAICT nothing except save some typing. I can remove it if you think.

Reply:
Yep seems strange and might make the code reader to ask the same thing I asked: Is there some additional macro magic hidden somewhere?